src/Languages/Editor/Test/LanguagesEditorMefCatalogFixture.cs

```diff
@@ -41,21 +37,18 @@ public class LanguagesEditorMefCatalogFixture : AssemblyMefCatalogFixture {
             };
         }
 
-        public virtual IExportProvider Create(CoreServicesFixture coreServices) => new LanguagesEditorTestExportProvider(CreateContainer(), coreServices);
+        public virtual IExportProvider Create() => new LanguagesEditorTestExportProvider(CreateContainer());
 
         protected class LanguagesEditorTestExportProvider : TestExportProvider {
-            private readonly CoreServicesFixture _coreServices;
-            public LanguagesEditorTestExportProvider(CompositionContainer compositionContainer, CoreServicesFixture coreServices) : base(compositionContainer) {
-                _coreServices = coreServices;
+            private readonly ICoreShell _coreShell;
+            public LanguagesEditorTestExportProvider(CompositionContainer compositionContainer) : base(compositionContainer) {
+                _coreShell = new TestCoreShell(new TestCompositionCatalog(compositionContainer));
             }
 
             public override Task<Task<RunSummary>> InitializeAsync(ITestInput testInput, IMessageBus messageBus) {
-                var editorShell = new TestEditorShell(CompositionContainer, _coreServices);
                 var batch = new CompositionBatch()
                     .AddValue(FileSystemStubFactory.CreateDefault())
-                    .AddValue<ICoreShell>(editorShell)
-                    .AddValue<IEditorShell>(editorShell)
-                    .AddValue(editorShell);
+                    .AddValue(_coreShell);
```

Comment:
Let's make `TestCoreShell _coreShell` and 
```cs
var batch = new CompositionBatch()
    .AddValue(FileSystemStubFactory.CreateDefault())
    .AddValue<ICoreShell>(_coreShell);
    .AddValue(_coreShell);
```

Direct reference to `TestCoreShell` can be used in tests